samples/webview/webview.cpp

```diff
@@ -224,7 +225,7 @@ bool WebApp::OnInit()
 
     // Create the memory files
     wxImage::AddHandler(new wxPNGHandler);
-    wxMemoryFSHandler::AddFile("logo.png", 
+    wxMemoryFSHandler::AddFile("logo.png",
```

Comment:
This is minor, but please avoid whitespace-only changes with the significant ones. TIA!